### PR TITLE
Update EntityAggregator.php

### PR DESCRIPTION
**Pull request by wannevancamp**
Fix typos

### 5. Checklist

- [ ] I have rebased my changes to remove merge conflicts
- [ ] I have written tests and verified that they fail without my change
- [ ] I have created a [changelog file](https://github.com/shopware/platform/blob/trunk/adr/workflow/2020-08-03-implement-New-Changelog.md) with all necessary information about my changes
- [ ] I have written or adjusted the documentation according to my changes
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [ ] I have read the contribution requirements and fulfil them.

<a href="https://gitpod.io/#https://github.com/shopware/platform/pull/2815"></a>

### CODE REVIEW

File: src/Core/Framework/DataAbstractionLayer/Dbal/EntityAggregator.php [hunk 1]
```diff
@@ -137,7 +137,7 @@ private function fetchAggregation(Aggregation $aggregation, EntityDefinition $de
             $clone->setTerm(null);
         }
 
-        $scoreCritera = clone $clone;
+        $scoreCriteria = clone $clone;
```
Comment (trandangtri) [inline src/Core/Framework/DataAbstractionLayer/Dbal/EntityAggregator.php:140]: Thanks for good catch, @wannevancamp  

I'm rather greedy :) We have some similar in other places. You don't mind if you help to fix it as well in a single MR?

src/Core/Checkout/Test/Cart/Order/RecalculationServiceTest.php
src/Core/Content/Test/Product/Repository/ProductRepositoryTest.php
src/Core/Framework/DataAbstractionLayer/Dbal/EntityAggregator.php

Thanks in advance.

**Review by trandangtri — CHANGES_REQUESTED** [review]

File: src/Core/Framework/DataAbstractionLayer/Dbal/EntityAggregator.php (hunk 1 above)
Comment (wannevancamp) [reply src/Core/Framework/DataAbstractionLayer/Dbal/EntityAggregator.php:140]: @trandangtri I've fixed the typos in the other files too.

**Comment by wannevancamp** [comment]
@trandangtri I've fixed the typos in the other files too.

**Comment by wannevancamp** [comment]
@trandangtri , is there anything I can do to get this merged?

**Comment by keulinho** [comment]
@wannevancamp sorry for the delay here, but from your side everything is fine, we are taking over from here.

**Comment by ghost** [comment]
Hey @wannevancamp  thanks again for your contribution, your changes are merged! :tada: :blue_heart:

**Comment by keulinho** [comment]
Here's the commit: https://github.com/shopware/platform/commit/9cb69c7228561270ac7eec4f6e124ebd3ef18ef3

**Pull request closed without merging**